HWAlevel/HW4/Module2_HW4/Program/Vegetables/Vegatables.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Module2_HW4.Program;
+
+namespace Module2_HW4.Program.Vegetables
+{
+    public abstract class Vegatables 
+    {
+        public string Name { get; set; }
+        public int Gram { get; set; }
+
+        public int resultGram = 0;
+
+        public const int gramPellPepper = 45;
```

Comment:
I think it's better to add classes of a specific vegetables that will inherit Vegatable